pravega/bookkeeper-operator
# Issue 9: Updating go version to 1.13 and using go modules

Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

### Change log description
Updated go version to `1.13.8` that uses GO modules. This will allow users to build code outside `GOPATH`

### Purpose of the change
Fixes #9 

### How to verify it
- All the make commands in the Makefile should work as expected.
- Should be able to build code outside $GOPATH.
- All basic operator functionality - cluster creation, deletion, scaling-up & down and upgrade should work as expected.
